### PR TITLE
feat: add image.digest value to deployment and cronjob

### PR DESCRIPTION
**Pull request by aslafy-z**
Add an optional image.digest field to Deployment and CronJob resources.

### CODE REVIEW

**Comment by stakater-user** [comment]
@aslafy-z Validation Successful

**Comment by rasheedamir** [comment]
@aslafy-z can you plz add description to the PR as well? And great thanks for the PR

**Review by rasheedamir — CHANGES_REQUESTED** [review]
- Update the readme
- Also add changelog in the readme in the end
- Add description to the PR

**Review by rasheedamir — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -415,6 +416,7 @@ Job paramater for each cronjob object at `cronJob.jobs`
 | `<name>.schedule`                  | Schedule of cronjob                                                                          | 
 | `<name>.image.repository`          | Repository of container image of cronjob                                                     |
 | `<name>.image.tag`                 | Tag of container image of cronjob                                                            |
+| `<name>.image.tag`                 | Digest of container image of cronjob                                                         |
```
Comment (rasheedamir) [inline README.md:419]: @aslafy-z plz change it to digest; it says tag

**Review by rasheedamir — CHANGES_REQUESTED** [review]
Please check; there is one issue in the readme

File: README.md [hunk 2]
```diff
@@ -567,6 +569,9 @@ To disable liveness or readiness probe, set value of `enabled:` to `false`.
 
 All notable changes to this project will be documented here
 
+### v1.3.0
```
Comment (rasheedamir) [inline README.md:572]: @aslafy-z Why should we bump minor? Wouldn't bumping the patch version will be enough? v1.2.8

Comment (aslafy-z) [inline README.md:572]:
```suggestion
### v1.2.8
```

Comment (aslafy-z) [reply README.md]: Done

**Comment by rasheedamir** [comment]
@aslafy-z can you plz fetch changes from main branch?

**Comment by aslafy-z** [comment]
@rasheedamir done :)

**Comment by hanzala1234** [comment]
@aslafy-z  digest can be added as tag in values.yaml. what is the use case of having this as separate value?

**Comment by aslafy-z** [comment]
@hanzala1234 I made a mistake, it's now fixed. `@` should be used as a separator instead of `:` as per https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy. Please take another look.

**Review by hanzala1234 — APPROVED** [review]

**Comment by rasheedamir** [comment]
@aslafy-z can you plz update the changelog? my apologies for delay; missed the PR

**Comment by rasheedamir** [comment]
@aslafy-z can you plz resolve merge conflict and change version to 1.2.10

**Comment by aslafy-z** [comment]
@rasheedamir Done

**Review by rasheedamir — APPROVED** [review]

**Comment by rasheedamir** [comment]
thank you @aslafy-z for the PR! looking forward to more PRs

**Pull request merged**